GCC 8.3.0 warning in copybw.cpp

```
g++ -O2 -I /usr/local/cuda-10.2/include -I ../include -I ../src -I /usr/local/cuda-10.2/include  -c -o copybw.o copybw.cpp
copybw.cpp:50:28: warning: ISO C++ forbids declaration of ‘main’ with no type [-Wreturn-type]
 main(int argc, char *argv[])
                            ^
```

```
$ gcc --version
gcc (Ubuntu 8.3.0-6ubuntu1) 8.3.0
Copyright (C) 2018 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

--- a/tests/copybw.cpp
+++ b/tests/copybw.cpp
@@ -47,7 +47,7 @@ using namespace gdrcopy::test;
 int num_write_iters = 10000;
 int num_read_iters  = 100;
 
-main(int argc, char *argv[])
+int main(int argc, char *argv[])
 {
     size_t _size = 128*1024;
     size_t copy_size = 0;
@@ -230,6 +230,8 @@ main(int argc, char *argv[])
     ASSERTDRV(gpuMemFree(d_A));
 
     ASSERTDRV(cuDevicePrimaryCtxRelease(dev));
+
+    return 0;
 }
 
 /*

